debian/libpam-policycache.postinst

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+DEFAULT_CACHE_STORAGE='/var/cache/libpam-policycache'
+
+case "$1" in
+  configure)
+    chmod 0700 "$DEFAULT_CACHE_STORAGE"
+    chown root:root /usr/bin/pam-escalate-helper
+    chmod 6755 /usr/bin/pam-escalate-helper
```

Comment:
It is quite unusual to have a binary be both setuid and setgid (all binaries I have are either or). Is there a reason for that?

In theory the chown is also unnecessary, but I can see why you'd want that for clarity.